api/src/main/resources/underlays/sd.yaml

```diff
@@ -0,0 +1,2604 @@
+# An initial OMOP underlay schema for development.
```

Comment:
nit: I'm guessing this was copy pasted from the other underlay, if so probably drop